src/main/java/com/toofifty/easyblastfurnace/utils/Strings.java

```diff
@@ -0,0 +1,68 @@
+package com.toofifty.easyblastfurnace.utils;
+
+import lombok.Getter;
+
+public enum Strings {
+    // MethodSteps
+    FILLCOALBAG("Fill coal bag"),
+    REFILLCOALBAG("Refill coal bag"),
+    EMPTYCOALBAG("Empty coal bag"),
+    WITHDRAWCOALBAG("Withdraw coal bag"),
+    WITHDRAWCOAL("Withdraw coal"),
+    WITHDRAWGOLDORE("Withdraw gold ore"),
+    WITHDRAWIRONORE("Withdraw iron ore"),
+    WITHDRAWMITHRILORE("Withdraw mithril ore"),
+    WITHDRAWADAMANTITEORE("Withdraw adamantite ore"),
+    WITHDRAWRUNITEORE("Withdraw runite ore"),
+    WITHDRAWICEORSMITHSGLOVES("Withdraw ice gloves or smiths gloves (i)"),
+    EQUIPICEORSMITHSGLOVES("Equip ice gloves or smiths gloves (i)"),
+    WITHDRAWGOLDSMITHGAUNTLETS("Withdraw goldsmith gauntlets"),
+    EQUIPGOLDSMITHGAUNTLETS("Equip goldsmith gauntlets"),
+    WITHDRAWSMITHINGCAPE("Withdraw Smithing cape"),
+    EQUIPSMITHINGCAPE("Equip Smithing cape"),
+    DEPOSITINVENTORY("Deposit inventory"),
+    PUTONTOCONVEYORBELT("Put ore onto conveyor belt"),
+    OPENBANK("Open bank chest"),
+    COLLECTBARS("Collect bars"),
+    WAITFORBARS("Wait for bars to smelt"),
+    DRINKSTAMINA(null),
+
+    // Bars
+    ADAMANTITEHYBRID("Gold + adamantite bars"),
+    ADAMANTITE("Adamantite bars"),
+    GOLD("Gold bars"),
+    MITHRIL("Mithril bars"),
+    MITHRILHYBRID("Gold + mithril bars"),
+    RUNITE("Runite bars"),
+    RUNITEHYBRID("Gold + runite bars"),
+    STEEL("Steel bars"),
+
+    // Patterns
+    COAL_FULL("^The coal bag is full.$"),
+    COAL_EMPTY("^The coal bag is now empty.$"),
+
+    // Actions
+    FILL("Fill"),
+    EMPTY("Empty"),
+    DRINK("Drink"),
+
+    // Stamina
+    WITHDRAW_STAMINA_POTION1("Withdraw stamina potion"),
+    WITHDRAW_STAMINA_POTION2("Withdraw stamina potion"),
+    WITHDRAW_STAMINA_POTION3("Withdraw stamina potion"),
+    WITHDRAW_STAMINA_POTION4("Withdraw stamina potion"),
+    DRINK_STAMINA_POTION1("Drink stamina potion"),
+    DRINK_STAMINA_POTION2("Drink stamina potion"),
+    DRINK_STAMINA_POTION3("Drink stamina potion"),
+    DRINK_STAMINA_POTION4("Drink stamina potion"),
+    GET_MORE_STAMINA_POTIONS("Get more stamina potions! Check settings to disable this");
+
+
+    @Getter
+    private final String txt;
```

Comment:
Scared of the letter e? 😆 

I'm not much of a java guy but I don't think there's much benefit to using an enum here - a class with static consts will make usage a bit neater IMO

Reply:
Haha, yeah I'm not much of one myself but I'm doing my best! I'll make it a class with static consts

Reply:
Okay, so a reason I made it an enum was so I could use the names of the enum variables as well as the values, with lines such as `Strings.MITHRIL.name()`

I'm working out how to do something similar with a class and static consts, but it looks like it's going to be a little messy.

For example, this:

`@Test
    public void adamantiteBarMethod()
    {
        metalMethods(ItemID.ADAMANTITE_ORE, ItemID.ADAMANTITE_BAR, BarsOres.ADAMANTITE_BAR.name(), Strings.ADAMANTITE.name());
        assertEquals(Strings.WITHDRAWADAMANTITEORE.getTxt(), methodHandler.getStep().getTooltip());
    }`

becomes this if I make Strings a class with static consts:
`    @Test
    public void adamantiteBarMethod() throws NoSuchFieldException
    {
        metalMethods(ItemID.ADAMANTITE_ORE, ItemID.ADAMANTITE_BAR, BarsOres.ADAMANTITE_BAR.name(), Strings.class.getField("ADAMANTITE").getName());
        assertEquals(Strings.WITHDRAWADAMANTITEORE, methodHandler.getStep().getTooltip());
    }`

Reply:
```Java
assertEquals(Strings.valueOf(barMethod).getTxt(), methodHandler.getMethod().getName());
```

There's also this line where I get the value of a variable in Strings using the dynamic argument `barMethod`. This seems very difficult to do with a class.

Reply:
I rewrote the tests in the way you suggested, and that allowed me to make this a class without any issues.